ormconfig.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "name": "development",
+    "type": "postgres",
+    "host": "localhost",
+    "port": 5432,
+    "username": "admin",
+    "password": "admin123",
+    "database": "devc",
+    "synchronize": true,
+    "migrationsRun": true,
+    "logging": true,
+    "cache": {
+      "type": "redis",
+      "duration": 30000,
+      "options": {
+        "host": "localhost",
+        "port": 6379
+      }
+    },
+    "entities": ["dist/entity/*.js"],
+    "migrations": ["dist/migration/*.js"],
+    "cli": {
+      "entitiesDir": "src/entity",
```

Comment:
I suggest we stick with the `models` dir as the `entities` dir for now, we will do the refactor in a separate PR.

Reply:
cool